resources/metabase-plugin.yaml

```diff
@@ -26,7 +26,7 @@ driver:
       required: false
     - user
     - password
-  connection-properties-include-tunnel-config: false
+  connection-properties-include-tunnel-config: true
```

Comment:
We only need to enable this option, no other code is required.

Reply:
👍